lua/starfall/libs_sh/input.lua

```diff
@@ -311,10 +312,10 @@ function input_library.lockControls(enabled)
 	end
 
 	if enabled then
-		if lockedControlCooldown > CurTime() then
+		if lockedControlCooldown + inputLockCooldown:GetFloat() > CurTime() then
```

Comment:
Mind moving the addition back to line 317?

Reply:
Do you mean adding 2 line breaks?

Reply:
no I mean the `+ inputLockCooldown:GetFloat()` idk why you moved it

Reply:
Oh so it could be changed during execution. Say you accidentally added an extra 0 when setting the convar. If I moved it from what it is now, you'd have to wait that extra magnitude of 10 seconds even if you changed it.

Reply:
Ah ok then